### PR TITLE
Move some specs from remote spec table into local repo

### PR DESCRIPTION
**Pull request by ahelwer**
Also update remote spec table format.

Specs added:

- Murat's PlusCal version of Lamport & Gray's analysis of Paxos vs. Two-Phase Commit
- Voucher Transaction System
- Atomic Commitment Protocol
- Byzantine Paxos

### CODE REVIEW

File: .ciignore [hunk 1]
```diff
@@ -5,3 +5,8 @@
 # Example:
 specifications/doesnotexist
 
+# Ignore submodules to ensure deterministic local execution
```
Comment (lemmy) [inline .ciignore:8]: Can you please explain why they would introduce non-determinism?  Don't the submodule points to a particular commit?

Comment (ahelwer) [reply .ciignore:8]: Basically if some people have the repo checked out with submodules they'll generate a different manifest.json than if other people have the repo checked out without submodules.

Comment (lemmy) [reply .ciignore:8]: Is that a problem given that the authoritative manifest is generated by our GitHub action?

Comment (ahelwer) [reply .ciignore:8]: It isn't actually generated by the action, the action just validates that it's correct.

Comment (lemmy) [reply .ciignore:8]: Sounds good enough to catch it when somebody accidentally generates the manifest without the submodules m

Comment (ahelwer) [reply .ciignore:8]: I suppose it is, but putting these dirs in the .ciignore file saves whoever is trying to submit specs to the examples repo a CI round trip. It's easy to remove these if we want to add these dirs to the CI eventually, although at this point we literally do want to ignore them.

Comment (ahelwer) [reply .ciignore:8]: Ah you're talking about adding the submodule specs to the CI. Well, maybe. I will look into doing that after this work.

**Pull request merged**